### PR TITLE
ENH: Add MicroBadger Label Schema on master branch

### PR DESCRIPTION
**Pull request by MayeulChassagnard**
Ref Issue https://github.com/thewtex/docker-opengl/issues/7

_For the `docker-opengl` image based on debian/jessie_

### CODE REVIEW

**Comment by MayeulChassagnard** [comment]
Updated & Rebased [here](https://github.com/thewtex/docker-opengl/pull/11) https://github.com/thewtex/docker-opengl/pull/11

**Comment by MayeulChassagnard** [comment]
Updated & Rebased [here](https://github.com/thewtex/docker-opengl/pull/11) https://github.com/thewtex/docker-opengl/pull/11

**Comment by MayeulChassagnard** [comment]
Updated & rebased [here](https://github.com/thewtex/SlicerDocker/pull/18) https://github.com/thewtex/SlicerDocker/pull/18

File: Makefile [hunk 1]
```diff
@@ -4,15 +4,23 @@ ORG = thewtex
 # Name of image
 IMAGE = opengl
 
+# TAG of the image
+TAG = jessie-v1.0.0
```
Comment (thewtex) [inline Makefile]: Let's use `debian` for the tag. We have `centos` for the other branch (the Linux distribution name). We will reserve the `-v1.0.0` for when we want to tag a specific 1.0.0 version.

**Pull request merged**